metricbeat/etc/fields.yml

```diff
@@ -47,31 +47,31 @@
               type: keyword
               description: >
                 Apache hostname.
-            - name: totalAccesses
+            - name: total_accesses
               type: integer
               description: >
                 Total number of access requests.
-            - name: totalKBytes
+            - name: total_kbytes
               type: integer
               description: >
                 Total number of kilobytes served.
-            - name: reqPerSec
+            - name: requests_per_sec
               type: float
               description: >
                 Requests per second.
-            - name: bytesPerSec
+            - name: bytes_per_sec
               type: float
               description: >
                 Bytes per second.
-            - name: bytesPerReq
+            - name: bytes_per_request
               type: float
               description: >
                 Bytes per request.
-            - name: busyWorkers
+            - name: workers.busy
```

Comment:
Sorry, I just noticed that the generated template contains a sub-dictionary. Please ignore the question.